openlibrary/plugins/openlibrary/js/bulk-tagger/BulkTagger.js

```diff
@@ -516,14 +526,20 @@ export class BulkTagger {
         })
             .then(response => {
                 if (!response.ok) {
-                    new FadingToast('Batch subject update failed. Please try again in a few minutes.').show()
+                // Error handling
+                    this.isSubmitting = false;
+                    this.submitButton.disabled = false;
+                    this.submitButton.textContent = 'Submit';
+                    new FadingToast('Error submitting batch').show();
```

Comment:
```suggestion
                    new FadingToast('Batch subject update failed. Please try again in a few minutes.').show();
```
Why was this changed?  There was no mention of changing the toast message in #8651.